src/components/layout/footer/Footer.jsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const Footer = () => {
```

Comment:
why is this in this PR?

Reply:
I just wanted to change `index.jsx` to its true name.